utm.yaml

```diff
@@ -1317,6 +1271,213 @@ components:
+          description: The request (by this USS) and response associated with the error.
+        problem:
+          type: string
+          description: Human-readable description of the problem with the response.
+
+    ExchangeRecord:
+      type: object
+      description: |-
+        A record of data exchange (USS - USS or USS - DSS).
+      required:
+      - request_source
+      - exchange
+      properties:
+        request_source:
+          type: string
+          description: |-
+            A coded value that indicates the requesting USS: 'LOCAL' (this USS) or 'REMOTE' (peer USS)
+          enum:
+          - LOCAL
+          - REMOTE
+        exchange:
+          allOf:
+          - $ref: '#/components/schemas/RequestResponse'
+          description: |-
+            Time that the response was received or sent
+
+    PositionRecord:
+      type: object
+      description: |-
+        A record of vehicle telemetry information received by this USS (typically for conformance monitoring).
+      required:
+      - time
+      - telemetry
+      properties:
+        time:
+          allOf:
+          - $ref: '#/components/schemas/Time'
+          description: |-
+            Time that this position data was received by the (local) USS
+        telemetry:
+          $ref: '#/components/schemas/VehicleTelemetry'
+
+    OperationRecord:
+      type: object
+      description: |-
+        A record of a position data gathered through the course of a complete operation
+      required:
+      - positions
+      - operation_id
+      properties:
+        positions:
+          type: array
+          items:
+            $ref: '#/components/schemas/PositionRecord'
+          default: []
+        operation_id:
+          description: ID of the Operation associated with 'positions'
+          allOf:
+          - $ref: '#/components/schemas/EntityID'
+
+    PlanningRecord:
+      type: object
+      description: |-
+        A record of a single attempt to (successfully or unsuccessfully) create an Operational Intent.
+      required:
+      - time
+      - ovns
+      - operation_id
+      - problem
+      properties:
+        time:
+          allOf:
+          - $ref: '#/components/schemas/Time'
+          description: Time that this planning event occurred
+        ovns:
+          type: array
+          minItems: 1
+          items:
+            $ref: '#/components/schemas/EntityOVN'
+        operation_id:
+          description: ID of the Operation being planned
+          allOf:
+          - $ref: '#/components/schemas/EntityID'
+        problem:
+          type: string
+          description: |-
+            A free text description of the problem(s) encountered during this planning attempt.
+
+    UserNotificationRecord:
+      type: object
+      description: |-
+        User notification record.
+      required:
+      - triggering_event_time
+      - notification_time
+      - notification_details
```

Comment:
Let's make this optional as the USS can already render it effectively optional by providing a generic value like "Notified user of specified event"